app/instrumentation-tests/src/org/commcare/androidTests/LazyVideosTests.kt

```diff
@@ -93,14 +93,17 @@ class LazyVideosTests : BaseTest() {
         onView(withTagValue(CoreMatchers.`is`(R.drawable.update_download_icon))).check(matches(isDisplayed()))
         InstrumentationUtility.stubIntentWithAction(Intent.ACTION_SEND)
         onView(withTagValue(CoreMatchers.`is`(R.drawable.update_download_icon))).perform(click())
-        onView(withId(R.id.video_button)).perform(click())
+        InstrumentationUtility.sleep(1)
+        InstrumentationUtility.waitForView(withSubstring("Download started"))
```

Comment:
this should be immediate after click, wait before and after this should not be required.

Reply:
This video button will be available only when the download is completed. L104 is doing that task here. Once the download is complete we click on the video button and verify the intent. We don't need it here as we will be clicking on `R.drawable.update_download_icon` and start the download.

Reply:
I am talking about `InstrumentationUtility.sleep()`  statements, I don't think they should be required before and after this statement since we are already using `waitForView` here.